app/policies/person_policy.rb

```diff
@@ -1,2 +1,24 @@
 class PersonPolicy < ApplicationPolicy
+
+  # Make records assocaited with a user record read only except for admins or self.
+  def update?
+    user_is_record? || @record.has_system_access ? division_admin : division_member_or_admin
+  end
+
+  def update_access?
+    division_admin
+  end
+
+  def update_password?
+    user_is_record? || division_admin
+  end
+
+  private
+
+  def user_is_record?
+    @record.user && @user && @user.id != nil && @record.user.id == @user.id
+  end
+
+  class Scope < DivisionOwnedScope
+  end
```

Comment:
I'm surprised the person_spec is still passing here since it hasn't been updated. It should probably be updated to cover this new code.